modules/virt-virtual-gpus-config-overview.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-mediated-devices.adoc
+
+:_content-type: CONCEPT
+[id="configuration-overview_{context}"]
+= Configuration overview
+
+When configuring mediated devices, an administrator must:
+
+* Create the mediated devices.
+* Expose the mediated devices to the cluster.
+
+The `HyperConverged` CR includes APIs that accomplish both tasks:
+
+.Creating mediated devices
+
+[source,yaml]
+----
+...
+spec:
+  mediatedDevicesConfiguration:
+    mediatedDevicesTypes: <.>
+    - <device_type>
+    nodeMediatedDeviceTypes: <.>
+    - mediatedDevicesTypes: <.>
+      - <device_type>
+      nodeSelector: <.>
+        <node_selector_key>: <node_selector_value>
+...
+----
+<.> Required: Configures global settings for the cluster.
+<.> Optional: Overrides the global configuration for a specific node or group of nodes. Must be used with the global `mediatedDevicesTypes` configuration.
+<.> Required if you use `nodeMediatedDeviceTypes`. Overrides the global `mediatedDevicesTypes` configuration for select nodes.
+<.> Required if you use `nodeMediatedDeviceTypes`. Must include a `key:value` pair.
+
+.Exposing mediated devices to the cluster
+
+[source,yaml]
+----
+...
+  permittedHostDevices:
+    mediatedDevices:
+    - mdevNameSelector: GRID T4-2Q <.>
+      resourceName: nvidia.com/GRID_T4-2Q
```

Comment:
The `resourceName`  here should match  the allocate on the node  - which can be different from the GPU name
```
oc get $NODE -o json  | jq '.status.allocatable | with_entries(select(.key | startswith("nvidia.com/"))) | with_entries(select(.value != "0"))'
```

This should be added via GPU operator  ( vgpu manager )

Reply:
@eifrach can I ask you to clarify what you mean by "This should be added via GPU operator ( vgpu manager )" an I missing some command here. Just picking this and reorganising

Reply:
they are using the Naming from the official [PCI database](https://pci-ids.ucw.cz/v2.2/pci.ids)
The name can be different from the device name shown in `lspci` and/or `nvidia-smi -L` 

the GPU operator adds the label to the node with the correct name 
you can get it with this command 

```console 
oc get $NODE -o json  | jq '.status.allocatable | with_entries(select(.key | startswith("nvidia.com/"))) | with_entries(select(.value != "0"))'
``` 

this  example is from Nvidia documentation 
```console
# using nvidia-smi to check what kind of GPU 
$ nvidia-smi -L
GPU 0: NVIDIA A10 (UUID: GPU-ebd34bdf-1083-eaac-2aff-4b71a022f9bd)
GPU 1: NVIDIA A10 (UUID: GPU-1795e88b-3395-b27b-dad8-0488474eec0c)


# when checking the node - the name is a bit different 
$ oc get node cnt-server-2 -o json | jq '.status.allocatable | with_entries(select(.key | startswith("nvidia.com/"))) | with_entries(select(.value != "0"))'
{
  "nvidia.com/NVIDIA_A10-12Q": "4"
}
```

Reply:
Thanks @eifrach  so as you state they should verify it with the command `oc get $NODE -o json  | jq '.status.allocatable | with_entries(select(.key | startswith("nvidia.com/"))) | with_entries(select(.value != "0"))'` and update it accordingly - correct ?.

Reply:
correct 
also i created an Ansible that automate the deployment of GPU operator and vGPU 
https://github.com/eifrach/openshift-virtualization-vGPU